environment-model/environment-model_ontology.ttl

```diff
@@ -4,14 +4,15 @@
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
```

Comment:
environment-model_instance.json is missing.

Reply:
add environment-model-instance.json. done!